### ISSUE TITLE
Reorganizing categories

### ISSUE THREAD

**Issue by XylotrupesGideon** [duerrsimon/bioicons#168]
I would suggest a reorganization of the categories to sort out overlaps and redundancies:

After looking through the different categories I came up with the following categories that would most likely cover all the current icons quite well, i am opening this to discussion to see if I have overlooked something. 
The reorganization will be some effort so I am happy to do it myself if we agree on categories and definitions.

## Animals
icons of complete animals

## Plants Algae
icons of complete plants, algae and complete plant parts

## Microbiology
bacteria, viruses, archaea, unicellular eukaryotes , parasites

## Tissues and Organs
Icons of specific tissues and organs of plants and animals

## Cell Building Blocks
icons of cell membranes organelles, membrane proteins for people to build their own cells

## Cell Types
icons of specific cell types e.g. myocytes, neurons, stem cells, plant cells ect.

## Oncology | Diseases
Cancer and other disease related assets

##  DNA | RNA | Nucleic Acids
everything related to nucleic acids, oligos, chromosomes

## Proteins | Amino Acids
Proteins, oligos, amino acids

## Lab Equipment
Laboratory machines large equipment, smaller things like spatulas and pipettes

## Lab Consumables
pipette tips, plastic wares, tissue paper ect.

## Computers and Informatics
Hardware, algorithms, Chemo and bioinformatic representations

## Imaging
microscopes, lenses, optics

## Graphs
all scientific graphs

## Procedures | Experiments | Workflows
Workflow overviews, experimental setups

## Symbols
Safety symbols, shapes, arrows

## People and institutions
People, people interactions, icons of buildings (university, institute)

## General items
Stuff that is not specific to science (food, calculators ect.)

**Comment by duerrsimon** [comment]
Thanks for the suggestions. I think we should do this once the new backend is up and running since then each illustration can also be in multiple categories and also can get some tags. There we have to invest effort for categorization anyway so it seems prudent to not do it twice. I'll be happy to include you in the categorization effort. 

The idea would be that users can suggest edits or new tags from the website which would then create a pull request in the new repo that will just contain the icon library in `bioicons/icons`

**Comment by duerrsimon** [comment]
>Is it possible to do category renaming but just renaming the applicable folders in the static icons or is there something else tha tneeds to be changes for new categories to show up on the website?

Yes currently the JSON that contains all icons which is used to render the page is built from the directory tree in `static/icons`

**Comment by XylotrupesGideon** [comment]
Good point to wait for the general update
